### PR TITLE
Elion treatment dict 223

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: gdcdictionary/schemas/treatment.yaml [hunk 1]
```diff
@@ -177,6 +179,126 @@ properties:
       - A081801
       - E4512
       - EA5142
+      - "901"
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:182]: do the new protocol_identifier enums need to be sorted alphabetically together with the existing ones?

Comment (bullenc) [reply gdcdictionary/schemas/treatment.yaml:178]: yes the script should do that... but it is having growing pains and not doing that... thanks for pointing out! alphabetized/numerically ordered 👍

File: gdcdictionary/schemas/treatment.yaml [hunk 2]
```diff
@@ -215,6 +345,20 @@ properties:
         - Oral
         - Not Reported
 
+  timepoint_category:
+    enum:
```
[the remark this answers is not in the document]
Comment (bullenc) [reply gdcdictionary/schemas/treatment.yaml:349]: done 👍

**Comment by bullenc** [comment]
> I didn't spot these changes that are listed in the jira:
> 
> * adding "TLR9 Agonist SD-101" for therapeutic_agent
> * deprecation of treatment_type: Surgery
> * removal of treatment_type: Cryoablation (but there seems to be a discussion between Bill and Tara on this in the Jira)
> 
> And generally, I think enums are supposed to be arranged alphabetically (alphanumerically?) with Not Reported and Unknown at the bottom. Other than the single comments I left, it looks like enums added to these two properties need to be slotted in the right place in the enum lists:
> 
> * treatment_outcome:
> * treatment_intent_type:

did more ordering, also thanks for flagging missing new enum, there was a typo in the sheet used to generate YAMLs and that got missed. Cryoablation is getting deprecated for migration after Elion, then will be removed. surgery was also deprecated for treatment_type.

File: gdcdictionary/schemas/treatment.yaml [hunk 3]
```diff
@@ -520,6 +664,7 @@ properties:
       - Wrist
       - Unknown
       - Not Reported
+      - Locoregional Site
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:667]: prob. need to move this one upwards for alphabeticalization

Comment (bullenc) [reply gdcdictionary/schemas/treatment.yaml:667]: darn how did I miss this one and some others 😢 ok fixed, thanks for the check.

**Review by wteouchicago — APPROVED** [review]

File: gdcdictionary/schemas/treatment.yaml [hunk 4]
```diff
@@ -193,6 +315,14 @@ properties:
         $ref:
           - "_terms_enum.yaml#/other/common"
 
+  residual_disease:
```
Comment (mpsolano) [inline gdcdictionary/schemas/treatment.yaml:318]: Should this and the other new properties have a `$ref`?

Comment (wwysoc2) [reply gdcdictionary/schemas/treatment.yaml:318]: They should, this should be resolved now.

**Review by bullenc — COMMENTED** [review]
looks good to me 👍

**Review by mpsolano — APPROVED** [review]

**Review by tlicht3 — APPROVED** [review]

**Pull request merged**